### PR TITLE
#166287132 Fix profile dropdown not showing

### PR DESCRIPTION
**Pull request by phemmelliot**
#### What Does This PR Do?
    This PR fixes the profile dropdown that is not showing

#### Description Of Task To Be Completed
    - add onclickoutside event listeners via package
    - change profile overflow to show

#### Any Background Context You Want To Provide?
   N/A
#### How can this be manually tested?
  Checkout to bg-fix-profile-dropdown-not-showing-166287132, click on the profile dropdown, you should see the profile

#### What are the relevant pivotal tracker stories?
[#166287132](https://www.pivotaltracker.com/story/show/166287132)

#### Screenshot(s)
N/A

### CODE REVIEW

File: client/assets/components/_dropdown.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import '../tools/functions.scss';
 
 .dropdown {
-  overflow: auto;
+  overflow: show;
```
Comment (olusoladavid) [inline client/assets/components/_dropdown.scss]: `show` doesn't seem to be a valid value for the overflow property. I am looking at [here](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow). Do you have a source for this?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (olusoladavid) [inline package.json]: I am not sure why `react-onclickoutside` is the only dependency showing up in your package.json and why it is showing up at this time? We already had this package installed in the past.

File: yarn.lock [hunk 3]
```diff
@@ -2,3 +2,7 @@
 # yarn lockfile v1
 
 
+react-onclickoutside@^6.8.0:
```
Comment (olusoladavid) [inline yarn.lock]: I think we can safely ignore this too.

**Review by olusoladavid — COMMENTED** [review]
Very neat job on this fix @phemmelliot. Would be nice to also add some hover style to the items so that a user knows which item is active when they hover on the dropdown. I made some other comments. Please check them out. Thanks. 🥂

File: yarn.lock (hunk 3 above)
Comment (phemmelliot) [reply yarn.lock]: Okay I will remove this

File: client/assets/components/_dropdown.scss (hunk 1 above)
Comment (phemmelliot) [reply client/assets/components/_dropdown.scss]: Thanks for pointing this out, I will change it to visible

**Review by Alameen688 — COMMENTED** [review]
Nice work 🎉 @phemmelliot. However, clicking outside anywhere in the document body toggles the dropdown. Kindly help look into that. Thanks.

**Comment by phemmelliot** [comment]
@Alameen688 closing the modal when you click outside of it is an expected behavior. When you click outside of the modal you are in essence saying that you don't want the modal to be in focus again

**Comment by Alameen688** [comment]
> @Alameen688 closing the modal when you click outside of it is an expected behavior. When you click outside of the modal you are in essence saying that you don't want the modal to be in focus again

@phemmelliot True. However, it doesn't just close the dropdown, it also opens it. That's why I said it "toggles" the dropdown. I think it's something we need to take a look at. Thank you.

**Comment by brianbirir** [comment]
@phemmelliot please see the attached gif. What @Alameen688 is saying is that if you click any where on the page, the dropdown menu appears.
drop-down

**Review by Alameen688 — APPROVED** [review]

**Pull request merged**